keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/AgentLockRepositoryTests.kt

```diff
@@ -1,6 +1,6 @@
 package com.netflix.spinnaker.keel.persistence
 
-import com.netflix.spinnaker.time.MutableClock
+import com.netflix.spinnaker.keel.time.MutableClock
```

Comment:
Could we even move `MutableClock` to kork? The reason I initially put it outside the `keel` package hierarchy is that it's not something directly to do with Keel.

Reply:
oh what do you know, it's already in kork: https://github.com/spinnaker/kork/blob/master/kork-test/src/main/java/com/netflix/spinnaker/kork/test/time/MutableClock.java

Reply:
😮

Reply:
However, I'm not going to update this PR to use the kork version :D sorry. I've backed out all the changes to the mutable clock class because the weren't needed

Reply:
Oh lord, is it different?

Reply:
I think there are some added extension functions in the keel one (like 'tick' vs increment)